dspace-server-webapp/src/test/java/org/dspace/app/rest/builder/BundleBuilder.java

```diff
@@ -51,7 +53,15 @@ public BundleBuilder withBitstream(Bitstream bitstream) {
     }
 
     public void cleanup() throws Exception {
-        delete(bundle);
+        try (Context c = new Context()) {
+            c.turnOffAuthorisationSystem();
+            // Ensure object and any related objects are reloaded before checking to see what needs cleanup
+            bundle = c.reloadEntity(bundle);
+            if (bundle != null) {
+                delete(c, bundle);
+                c.complete();
```

Comment:
seems a risk to keep the context open, there are more such use cases below

Reply:
it uses the try-with-resource syntax (it is used also in other recent part of the sourcecode)
https://docs.oracle.com/javase/tutorial/essential/exceptions/tryResourceClose.html